pkg/providers/vsphere/virtualmachine/backup_test.go

```diff
@@ -689,9 +689,10 @@ func backupTests() {
 
 						diskData := []backupapi.PVCDiskData{
 							{
-								FileName:    vcSimDiskFileName,
+								FileName:    vcSimDiskFileName, //
```

Comment:
Nit, why the empty comment? No reason to re-spin the PR, just curious.

Reply:
That was a leftover.  Removed.